Build broken on 1.2.2

## Description
Build is not working in prod right now:
```
[2021-05-26 19:38:25 +0000] [7] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/wps/thunderbird/__init__.py", line 7, in <module>
    from .wsgi import application  # noqa: F401
  File "/opt/wps/thunderbird/wsgi.py", line 4, in <module>
    from .processes import processes
  File "/opt/wps/thunderbird/processes/__init__.py", line 1, in <module>
    from .wps_generate_climos import GenerateClimos
  File "/opt/wps/thunderbird/processes/wps_generate_climos.py", line 13, in <module>
    from dp.generate_climos import generate_climos, dry_run_handler, input_check
  File "/usr/local/lib/python3.9/site-packages/dp/generate_climos.py", line 39, in <module>
    cdo = Cdo()
  File "/usr/local/lib/python3.9/site-packages/cdo.py", line 144, in __init__
    self.operators         = self.__getOperators()
  File "/usr/local/lib/python3.9/site-packages/cdo.py", line 190, in __getOperators
    version = parse_version(getCdoVersion(self.CDO))
  File "/usr/local/lib/python3.9/site-packages/cdo.py", line 67, in getCdoVersion
    proc = subprocess.Popen([path2cdo, '-V'], stderr=subprocess.PIPE, stdout=subprocess.PIPE)
  File "/usr/local/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/local/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'cdo'
[2021-05-26 19:38:25 +0000] [7] [INFO] Worker exiting (pid: 7)
[2021-05-26 19:38:26 +0000] [1] [INFO] Shutting down: Master
[2021-05-26 19:38:26 +0000] [1] [INFO] Reason: Worker failed to boot.
```

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,1 +1,27 @@
+# Git
 .git
+.github
+.gitignore
+
+# Build
+.pytest_cache
+
+# Config and Docs 
+dev-component/
+docs/
+etc/
+pavics-compnent/
+.dockerignore
+LICENSE.txt
+Makefile
+Dockerfile
+AUTHORS.md
+
+# Tests
+tests/
+
+# Ignore files
+*.json
+*.yml
+*.in
+*.ini

--- a/.github/workflows/anchore-analysis.yml
+++ b/.github/workflows/anchore-analysis.yml
@@ -1,10 +1,10 @@
 name: Anchore Container Scan
-on:
-  push:
-    branches:
-      - master
-      - '*security*'
-      - '*vulnerability*'
+on: [push]
+  # push:
+  #   branches:
+  #     - master
+  #     - '*security*'
+  #     - '*vulnerability*'
 
 jobs:
   Anchore-Build-Scan:

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,6 +1,6 @@
 FROM python:3.8-slim
 
-LABEL Maintainer=https://github.com/pacificclimate/thunderbird Description="thunderbird WPS" Vendor="pcic" Version="1.2.1"
+LABEL Maintainer=https://github.com/pacificclimate/thunderbird Description="thunderbird WPS" Vendor="pcic" Version="1.2.3"
 ENV DEBIAN_FRONTEND=noninteractive
 ENV PIP_INDEX_URL="https://pypi.pacificclimate.org/simple/"
 
@@ -17,6 +17,7 @@ RUN apt-get update && \
     # once 1.24.2 becomes available through the base image.
     # Issue link: https://github.com/advisories/GHSA-mh33-7rrq-662w
     apt-get remove -y python3-urllib3 && \
+    pip install -U pip && \
     pip install . && \
     pip install gunicorn
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,45 +1,24 @@
-# vim:set ft=dockerfile:
-FROM python:3.9-slim
-MAINTAINER https://github.com/pacificclimate/thunderbird
-LABEL Description="thunderbird WPS" Vendor="pcic" Version="1.2.2"
+FROM python:3.8-slim
 
+LABEL Maintainer=https://github.com/pacificclimate/thunderbird Description="thunderbird WPS" Vendor="pcic" Version="1.2.1"
 ENV DEBIAN_FRONTEND=noninteractive
 ENV PIP_INDEX_URL="https://pypi.pacificclimate.org/simple/"
 
-# Update Debian system
-RUN apt-get update && apt-get install -y \
-    build-essential \
-    cdo \
-    git \
-    # HDF5 libraries for cdo
-    libhdf5-serial-dev \
-    netcdf-bin \
-    libnetcdf-dev && \
+COPY . /tmp
+WORKDIR /tmp
+
+RUN apt-get update && \
+    apt-get install -y build-essential cdo libhdf5-serial-dev netcdf-bin libnetcdf-dev && \
     rm -rf /var/lib/apt/lists/* && \
-    # Combats the issue found here:
+    # this line combats the issue found here:
     # https://superuser.com/questions/1347723/arch-on-wsl-libqt5core-so-5-not-found-despite-being-installed
     strip --remove-section=.note.ABI-tag /usr/lib/x86_64-linux-gnu/libQt5Core.so.5 && \
     # Removing this package prevents a scan failure. It will be reintroduced
     # once 1.24.2 becomes available through the base image.
     # Issue link: https://github.com/advisories/GHSA-mh33-7rrq-662w
-    apt-get remove -y python3-urllib3 python3-pil python3-pandas libmariadb3 && \
-    # Removing this metadata helps prevent issues from vulnerability scan
-    rm -f /usr/local/lib/python3.9/site-packages/pip-21.0.1.dist-info/METADATA
-
-
-COPY . /opt/wps
-
-WORKDIR /opt/wps
-
-# Install WPS
-RUN pip install . && \
+    apt-get remove -y python3-urllib3 && \
+    pip install . && \
     pip install gunicorn
 
-# Start WPS service on port 5000 on 0.0.0.0
 EXPOSE 5000
 CMD ["gunicorn", "--bind=0.0.0.0:5000", "thunderbird.wsgi:application"]
-
-# docker build -t pacificclimate/thunderbird .
-# docker run -p 5000:5000 pacificclimate/thunderbird
-# http://localhost:5000/wps?request=GetCapabilities&service=WPS
-# http://localhost:5000/wps?request=DescribeProcess&service=WPS&identifier=all&version=1.0.0

--- a/requirements.txt
+++ b/requirements.txt
@@ -9,4 +9,4 @@ xarray==0.15.1
 nchelpers==5.5.7
 werkzeug==1.0.1
 wps-tools==2.0.0
-cdo==1.5.3
+cdo==1.5.4

